.pre-commit-config.yaml

```
@@ -52,6 +52,12 @@ repos:
         args: [--py37-plus]
         exclude: '^(install|get)-poetry.py$'
 
+  - repo: https://github.com/hadialqattan/pycln
+    rev: v1.2.2
+    hooks:
+      - id: pycln
+        args: [--config=pyproject.toml]
```

since this will likely only ever be used in `pre-commit`, it may be simpler just to inline the config here:
```suggestion
        args: [--all]
```

---

I made it inlined 👍🏻